tests/test_nc_to_mmd.py

```diff
@@ -1079,13 +1079,22 @@ def test_checksum(self):
         tested = tempfile.mkstemp()[1]
         fn = 'tests/data/reference_nc.nc'
         nc2mmd = Nc_to_mmd(fn, check_only=True)
-        nc2mmd.to_mmd()
+        nc2mmd.to_mmd(checksum_calculation=True)
         checksum = nc2mmd.metadata['storage_information']['checksum']
         with open(tested, 'w') as tt:
             tt.write('%s *%s'%(checksum, fn))
         md5hasher = FileHash('md5')
         self.assertTrue(md5hasher.verify_checksums(tested)[0].hashes_match)
 
+    def test_checksum_off(self):
+        """ToDo: Add docstring"""
+        fn = 'tests/data/reference_nc.nc'
+        nc2mmd = Nc_to_mmd(fn, check_only=True)
+        nc2mmd.to_mmd(checksum_calculation=False)
```

Comment:
should also be tested without input to ensure the default is correct..

Reply:
will do